src/components/PollViewer/PollDraftHandler.vue

```diff
@@ -0,0 +1,100 @@
+<!--
+  - SPDX-FileCopyrightText: 2024 Nextcloud GmbH and Nextcloud contributors
+  - SPDX-License-Identifier: AGPL-3.0-or-later
+-->
+
+<template>
+	<NcDialog class="drafts"
```

Comment:
Make it bigger so it can hide the other dialog? 
![image](https://github.com/user-attachments/assets/28df4c9e-6e59-4065-8402-cf65f01b6bcc)

Or close that modal once this is open. 
A good addition would be to rename the button to "Back" instead of "Create a new poll" when it's opened from poll editor :).  The flow would make sense much better.

Reply:
I'd just hide the button in case PollEditor is open (as we can see it behind), but don't close it, like ConversationSettings dialog stays behind while you're checking bans - subfeature on top of main one.

Besides, PollEditor can be larger if there are more answers filled. Though we can squezee drafts dialog to 2 drafts in a row?

Reply:
I'd just hide the button in case PollEditor is open (as we can see it behind), but don't close it, like ConversationSettings dialog stays behind while you're checking bans - subfeature on top of main one.

Fine by me, but I'd still prefer renaming it to "back"

> Besides, PollEditor can be larger if there are more answers filled. Though we can squezee drafts dialog to 2 drafts in a row?

Yes, but we should cover all scenarios. It needs to have a default height and width that can cover the poll editor ?

Reply:
Hid the button for now. Size issue can be solved in follow-up (I thought of extending PollEditor to normal size, maybe we'll shrink drafts to 1-2 row with that change)